Makefile

```diff
@@ -24,7 +24,7 @@ TBB_BASE    :=
 CUDA_BASE   := /usr/local/cuda
 ALPAKA_BASE := /usr/local/alpaka/alpaka
 CUPLA_BASE  := /usr/local/alpaka/cupla
-ONEAPI_BASE := /opt/intel/inteloneapi/compiler/latest/linux
+SYCL_BASE := /data/user/fwyzard/sycl/build
```

Comment:
Should this be something along `/usr/local/sycl/build` to stay more generic?

Reply:
Ops, that sneaked in... I'll change it to something more generic, like `/opt/sycl/latest`.